src/schema/users.schema.ts

```diff
@@ -25,8 +24,7 @@ export class UserSchema {
   @ApiProperty({ type: String, description: 'User name' })
   username: string;
 
-  @prop({ required: true })
-  @Exclude()
+  @prop({ required: true, select: false })
   password: string;
```

Comment:
Missing @ApiProperty find `password`

Reply:
password is exclude since query from database have only `userService.getUserByUsername` that return password for signin validation.

Reply:
Wrong Spell